packages/executor/src/utils/getGasFee.ts

```diff
@@ -49,6 +49,9 @@ export const getGasFee = async (
 };
 
 function parseGwei(num: number): BigNumber {
+  if (typeof num !== "number") {
```

Comment:
What is the type that we are receiving?

Reply:
sometimes it is `string`, sometimes it is `number`